cmd/textile/threads.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+	"github.com/textileio/go-threads/core/thread"
+	"github.com/textileio/textile/cmd"
+)
+
+func init() {
+	rootCmd.AddCommand(threadsCmd)
+	orgsCmd.AddCommand(lsThreadsCmd)
+}
+
+var threadsCmd = &cobra.Command{
+	Use: "threads",
```

Comment:
These are really DBs? If so, should we use "db" or something in the api and cli names?

Reply:
They could be DBs, but they could also be raw threads created with the Net API. Tricky...